src/App.js

```diff
@@ -15,6 +15,7 @@ const ROUTES = {
   SETTINGS: '/settings',
```

Comment:
seems like you want to remove settings here?

Reply:
yes